backend/Persistence/Repositories/MetadataRepository.cs

```diff
@@ -6,6 +6,7 @@
 using OpenData.Persistence.Contexts;
 using System.Web;
 using System;
+using System.Linq;
```

Comment:
?

Reply:
Ahhh, this is from when the comment had functions in the MetadataController.

Reply:
Fixed